pkgs/development/python-modules/3to2/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchurl
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "3to2";
+  version = "1.1.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
```

Comment:
I tried fetchPypi, but I it 404s for some reason, I think it has something to do with the package name starting with a number instead of a character.